include/particles/particle.h

```diff
@@ -161,6 +161,11 @@ class Particle : public ParticleBase<Tdim> {
   //! Return strain of the particle
   Eigen::Matrix<double, 6, 1> strain() const override { return strain_; }
 
+  //! Return shear strain of the particle
+  Eigen::Matrix<double, 3, 1> shear_strain() const override {
```

Comment:
Can we not just `tail` the output of `strain`?

Reply:
We can do that, so less functions. Shall I go ahead with this?